db/controllers/articles.controller.js

```diff
@@ -24,3 +24,21 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (typeof inc_votes !== "number") {
+    return res.status(400).send({ msg: "Bad request - invalid input" });
+  }
```

Comment:
You don't need to manually check this - if inc_votes is a not number psql will throw an error and you can catch that in your catch block!